libs/vm-modules/src/math/bignumber.cpp

```diff
@@ -406,6 +406,67 @@ bool UInt256Wrapper::IsGreaterThan(Ptr<Object> const &lhso, Ptr<Object> const &r
   return rhs->number_ < lhs->number_;
 }
 
+vm::ChargeAmount UInt256Wrapper::AddChargeEstimator(fetch::vm::Ptr<Object> const & /*lhso*/,
+                                                    fetch::vm::Ptr<Object> const & /*rhso*/)
+{
+  return 1;
```

Comment:
I would suggest a named constant private field instead, just for a tuning purpose later.